src/components/datepicker/DatePickerAgenda.vue

```diff
@@ -545,6 +548,20 @@ export default {
       width: get-size(desktop, width);
     }
 
+    &--rtl {
+      .datepicker-days .datepicker-day.first .datepicker-day__effect:before,
+      .datepicker-days .datepicker-day.select-start:hover:not(.selected) .datepicker-day__effect:before {
```

Comment:
maybe reduce those class selector

Reply:
Those are the generated selectors from dev tools for the LTR version. In order to ensure we get a proper RTL I used the same class selectors.